File: resources/[Addons]/bot_api/server/lua/commands/Actions/UnbanAC.lua
function _bot_api:UnbanAC(ban_id, cb)
    local succes = exports['WaveShield']:unbanPlayer(ban_id, "Unban", "From discord bot")

    if (succes) then
        cb("Le joueur avec le ban ID WaveShield "..ban_id.." est unban !")
    else
        cb("Ce ban ID WaveShield ne correspond pas !")
    end
end

File: resources/[Addons]/bot_api/server/commands/Actions/UnbanAC.js
module.exports = {
    name: "unbanwaveshield",
    description: "Unban Anti-Cheat",
    role: "modo",

    options: [
        {
            name: "ban_id",
            description: "Ban id du joueurs",
            required: true,
            type: "STRING",
        }
    ],

    run: async (client, interaction, args) => {
        var OneLifeClass = global.exports[`${GetCurrentResourceName()}`].GetClassbot_api()

        OneLifeClass.UnbanAC(null, args.ban_id, function(msg){
            const embed = new client.Embed()
                .setTitle(`Unban AC Joueur`)
                .setDescription(`${msg}`);
            interaction.reply({ embeds: [embed] });
        })
    },
};
